docs/ErDiagram.puml

```diff
@@ -0,0 +1,30 @@
+@startuml
+
+title Entity Relationship Diagram 
+' hide the spot
+hide circle
+
+' avoid problems with angled crows feet
+skinparam linetype ortho
+
+entity "Consent" as e01 {
+  * **id**
+  * initiator_id
```

Comment:
Can we add a field for the time the consent resource was created? Just for history...